pom.xml

```
@@ -66,7 +66,7 @@
     <dependency>
       <groupId>uk.gov.ons.ssdc</groupId>
       <artifactId>ssdc-rm-common-entity-model</artifactId>
-      <version>4.21.0</version>
+      <version>4.22.0-SNAPSHOT</version>
```

TODO: remove -SNAPSHOT before merging
```suggestion
      <version>4.22.0</version>
```